core/sass/euexit-form.scss

```diff
@@ -0,0 +1,30 @@
+@import './partials/base';
+@import './components/form-wizard';
+@import './components/breadcrumbs';
+
+#content { 
+  h1 {
+    margin-bottom: 20px;
+  }
+  .body-text {
+    margin-bottom: 50px;
+  }
+  .breadcrumbs {
+    border-bottom: 0;
+    padding-top: 0;
```

Comment:
don't override the padding pls or it looks weird :( getting rid of the border is fine though. if you're keeping the gradient background for the page you should also change background-color to transparent